tests/test_content.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from pypst import Document, Set
+from pypst.content import Content
+
+
+def test_empty_content():
+    obj = Content()
+    assert obj.render() == "#[]"
+
+
+def test_nested_content():
+    obj = Content(Content())
+    assert obj.render() == "#[]"
+
+    obj = Content(Content("Foo", context=True))
+    assert obj.render() == "#context [Foo]"
+
+    obj = Content(Content("Foo"), context=True)
+    assert obj.render() == "#context [Foo]"
```

Comment:
Could be simplified by using `pytest.mark.parametrize`, but that's mostly nitpicking.